fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java

```diff
@@ -164,24 +160,6 @@ public Statistics visitComparisonPredicate(ComparisonPredicate cp, EstimationCon
         }
         ColumnStatistic statsForLeft = ExpressionEstimation.estimate(left, context.statistics);
         ColumnStatistic statsForRight = ExpressionEstimation.estimate(right, context.statistics);
-        if (aggSlots != null) {
```

Comment:
means we give up the estimation of having?

Reply:
pls add stats. estimation behavior changes notes in "Proposed changes", including fixed parameter changing, etc.